src/main/resources/org/biouno/unochoice/CascadeChoiceParameter/index.jelly

```diff
@@ -2,6 +2,7 @@
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
   ${it.parameters.clear()}
   <st:include page="/org/biouno/unochoice/common/choiceParameterCommon.jelly"/>
+  <st:bind value="${it}" var="cascadeChoiceParameter"/>
```

Comment:
:+1: I think I used the other ugly syntax to fix warnings on my IDE (Eclipse + WebDev at that time, I think?). Do VSCode/WebStorm understand this? Maybe there's a stapler plug-in that I can install?

Reply:
I'm on IntelliJ and it seems to understand it.
Also, now most of the JS is out of this file, so there isn't much context left.
I do have this plugin - https://github.com/jenkinsci/idea-stapler-plugin

Reply:
https://github.com/jenkinsci/active-choices-plugin/pull/79#discussion_r1254815629

That's the reason I changed this.

Reply:
Right, inline syntax is not going to work with Content-Security-Policy prohibiting inline scripts. See [JENKINS-60865](https://issues.jenkins.io/browse/JENKINS-60865) for context.